x/evm/keeper/keeper.go

```diff
@@ -221,11 +221,11 @@ func (k *Keeper) SetHooks(eh types.EvmHooks) *Keeper {
 }
 
 // PostTxProcessing delegate the call to the hooks. If no hook has been registered, this function returns with a `nil` error
-func (k *Keeper) PostTxProcessing(ctx sdk.Context, from common.Address, to *common.Address, receipt *ethtypes.Receipt) error {
+func (k *Keeper) PostTxProcessing(ctx sdk.Context, msg ethtypes.Message, receipt *ethtypes.Receipt) error {
```

Comment:
nit, we should use `core.Message` since `ethtypes.Message` has the following comment:

```go
// Message is a fully derived transaction and implements core.Message
//
// NOTE: In a future PR this will be removed.
```

Reply:
done